tensorflow/lite/kernels/split.cc

```diff
@@ -53,6 +53,9 @@ TfLiteStatus ResizeOutputTensors(TfLiteContext* context, TfLiteNode* node,
     axis_value += NumDimensions(input);
   }
 
+  TF_LITE_ENSURE(context, axis_value >= 0);
+  TF_LITE_ENSURE(context, axis_value < NumDimensions(input));
```

Comment:
I think the axis value can be in the range of [-rank(value), rank(value)) ? Can you add code to handle in this case, or is it already handled elsewhere?

Reply:
@haozha111: as the axis is converted to positive in code above, so the negative bounds are eliminated.